napari/utils/events/evented_model.py

```diff
@@ -276,7 +282,59 @@ def _super_setattr_(self, name: str, value: Any) -> None:
         else:
             super().__setattr__(name, value)
 
+    def _check_if_differ(self, name, old_value) -> Tuple[bool, any]:
+        """
+        Check new value of a field and emit event if it is different from the old one.
+        Returns True if event was emitted, else False.
+        """
+        new_value = getattr(self, name, object())
+        if name in self.__eq_operators__:
+            are_equal = self.__eq_operators__[name]
+        else:
+            are_equal = pick_equality_operator(new_value)
+        if not are_equal(new_value, old_value):
+            return True, new_value
+        return False, None
```

Comment:
Optional: I don't think there's a disadvantage to return `new_value` and it makes the code more concise and readable.

```suggestion
        return not are_equal(new_value, old_value), new_value
```